src/python_testing/matter_testing_infrastructure/matter/testing/apps.py

```diff
@@ -216,3 +328,368 @@ def create_acl_entry(self, dev_ctrl: ChipDeviceController, provider_node_id: int
             nodeid=provider_node_id,
             attributes=[(0, acl_attribute)]
         )
+
+    async def set_acl_for_requestor(
+            self,
+            requestor_node: int,
+            provider_node: int,
+            fabric_index: int,
+            original_requestor_acls: list
```

Comment:
why are these passed in vs. kept as part of the class? The function description says this is read from the node. You need this later for cleanup too, right?